src/xcode/ENA/ENA/Source/Extensions/Foundation+Base64URL.swift

```diff
@@ -21,6 +21,11 @@ extension Data {
 }
 
 extension String {
+	
+	var isBase64Encoded: Bool {
+		return (self.contains("+") || self.contains("/") || self.contains("="))
```

Comment:
Hmm not sure if this valid. Should we just try to decode the string into Data and if this fails we can return false here?

Reply:
this is the weird part, base64 can be decoded using the base64URL class initializer and it actually works, but im not sure if it is working because of the mock QRCodes or whats so i thought better to just check first and decode accordingly

Reply:
Yeah but `YW55IGNhcm5hbCBwbGVhcw` should be valid Base64 and this function would say its not :/

Reply:
I copied the definition of a valid base64 from the techSpecs maybe it is wrong i will fix it then
https://github.com/corona-warn-app/cwa-app-tech-spec/blob/proposal/rapid-test/docs/spec/corona-test-client.md#qr-code-of-a-rapid-antigen-test
<img width="898" alt="Screenshot 2021-04-13 at 13 00 11" src="https://user-images.githubusercontent.com/15270737/114542416-324e9d00-9c58-11eb-9688-3ea541149ae0.png">